config/config.go

```diff
@@ -38,7 +38,7 @@ type MessageProtocolConfig struct {
 }
 type SourceSubnet struct {
 	SubnetID              string                           `mapstructure:"subnet-id" json:"subnet-id"`
-	ChainID               string                           `mapstructure:"chain-id" json:"chain-id"`
+	BlockchainID          string                           `mapstructure:"blockchain-id" json:"blockchain-id"`
```

Comment:
We'll have to update https://github.com/ava-labs/teleporter/blob/main/docker/relayerConfigTemplate.json to match the new config options. It looks like that slipped through the cracks of the other PR.